La'Tonia/buy_me_a_laptop_challenge.py

```diff
@@ -0,0 +1,168 @@
+macbook_dict = laptops[0]
+print(laptops[0]["url"])
+print()
+
+# TODO: Write code to print all possible prices of the 16-inch MacBook Pro
+# Tried various ways to add .join to prices to output $ at start of each price ... succeeded in adding $ at start of each character ... lol
+print(f'''Two Price Points for 16-inch MacBook Pro - that\'s a total of {len(laptops[0]["types"][1]["price"])}.''')
+print(f'Those Two Price Points: {laptops[0]["types"][1]["price"]}')
+print()
+
+# TODO: Write code to print all the color options for Dell XPS 13 
+# Instruction above likely means 13-inch
+# Printed all color options just in case
+print(f'Color Options for 13-inch Dell XPS: {laptops[1]["types"][0]["colors"]}')
+print(f'Color Options for 15-inch Dell XPS: {laptops[1]["types"][1]["colors"]}')
+print()
+
+# TODO: Write code to print the description of Dell XPS laptop
+print(laptops[1]["description"])
+print(f'Description for Dell XPS: {laptops[1]["description"]}')
+print()
+
+print("Question 2: Out of Stock laptops")
+print("Suppose that the 13-inch MacBook Pro in space gray color is sold out. Also, the same laptop with 1 TB storage is out of stock as well. Update the list of dictionaries such that these options are removed. Print the updated dictionary.")
+print()
+
+# TODO: Update the laptops dictionary
+# Easier to first isolate the targets in output then double back to (try to) delete
+# No 1TB in 13-inch MacBook option and modified this question to answer it using the 16-inch option which does have 1TB
+# Below len code per trying to figure out reasons two items in list but getting 'out of range' error when try to print
+print(f'Total Items in Colors List: {len(laptops[0]["types"][0]["colors"])}')
+print(f'13-inch MacBook Color Options: {laptops[0]["types"][0]["colors"]}')
+print(f'13-inch MacBook Color Option Need to Delete: {laptops[0]["types"][0]["colors"][0]}')
+laptops[0]["types"][0]["colors"][0]
+print()
+
+print(f'Total Items in Storage List: {len(laptops[0]["types"][1]["colors"])}')
+print(f'16-inch MacBook Storage Options: {laptops[0]["types"][1]["storage"]}')
+print(f'16-inch MacBook Storage Option Need to Delete: {laptops[0]["types"][1]["storage"][1]}')
+laptops[0]["types"][1]["storage"][1]
+print()
+
+# Below code starting on lines 150 and 154 to update dictionaries to fulfill this assignment exercise
+pprint(laptops[0]["types"][0]["colors"][0])
+print(laptops[0]["types"][1]["storage"][1])
+print()
+
+# Below code didn't return removal of specified item
+# laptops[0]["types"][0].pop(["colors"][0])
+# pprint(laptops)
+
+# # Below code throws error - index out of range, makes no sense to me given the current data structure ... What am I missing?
+# laptops[0]["types"][1].pop(["storage"][1])
```

Comment:
Refer to my simple code example in the group chat, I believe you can only pop or remove single elements at a time, instead of referencing an index+list combo or a dictionary+key combo